### DISCUSSION TITLE
PDF highlights across pages

### DISCUSSION THREAD

**Discussion by drvenom123** [koreader/koreader#10581] [Q&A] [answered]
Is there a way to highlight across Pages? If you come across a passage at the end of the page that extends onto the following page can you highlight a cross that page break to highlight the entire passage? Also, are there any plans for multi window support? For example, in samsung tablets you can use two apps at the same time, but koreader only works in full screen mode.

**Comment by pazos** [comment]
Highlight across page boundaries is not supported on fixed media (pdf, djvu...), IIRC

Multiwindow support for android is being tracked in https://github.com/koreader/android-luajit-launcher/issues/238. It should be way easier to implement now, since we're more compliant with the android framework now.

Still waiting for a volunteer :)

**Comment by hius07** [comment, marked as answer]
Yes, you can make multi-page highlight in cre and pdf documents.
Highlight the starting fragment (within one page), then choose "Select" in the popup dialog and then highlight the ending fragment (in any other page if you wish).
